com.br_automation.buoat.xddeditor/plugin.xml

```diff
@@ -23,4 +23,32 @@
       </parser>
    </extension>
 
+   <extension point="org.eclipse.emf.ecore.generated_package">
+      <!-- @generated XDD -->
+      <package
+            uri="http://www.ethernet-powerlink.org"
+            class="com.br_automation.buoat.xddeditor.XDD.XDDPackage"
+            genModel="model/XDD.genmodel"/>
+   </extension>
+
+   <extension point="org.eclipse.emf.ecore.extension_parser">
+      <!-- @generated XDD -->
+      <parser
+            type="xdd"
+            class="com.br_automation.buoat.xddeditor.XDD.util.XDDResourceFactoryImpl"/>
+   </extension>
+   <extension point="org.eclipse.emf.ecore.generated_package">
+      <!-- @generated Firmware -->
+      <package
+            uri="file:/D:/OCPV2.2_Release/openconfigurator-core-plugin/org.epsg.openconfigurator/resources/firmwareschema/fw_schema.xsd"

```

Comment:
Include the schema in the project and replace the locale path

Reply:
Fix provided in pull request #2